pypsa/io.py

```diff
@@ -328,24 +328,31 @@ def save_meta(self, meta):
             self.ds.attrs["meta"] = json.dumps(meta)
 
         def save_snapshots(self, snapshots):
-            snapshots.index.name = "snapshots"
+            snapshots = snapshots.rename_axis(index="snapshots")
```

Comment:
Ai, this can be a multi-index, too, can it not? @FabianHofmann

Reply:
Indeed it can be, but I think this should still be fine. Pandas multiindex have a name and level names. Can test it later though.